### DISCUSSION TITLE
CosmosOS with IronPython

### DISCUSSION THREAD

**Discussion by CaptainHaddock2016** [CosmosOS/Cosmos#2307] [Q&A]
How do I use CosmosOS with IronPython? It states on the cosmos os website that this is possible.

**Comment by zarlo** [comment]
you should be able to use ipyc to compile it to a .net dll

**Comment by quajak** [comment]
You will need to recreate the information which is currently in the csproj file in the ipython project. At that point simply compiling the program should trigger cosmos.
